src/js/pages/SundarGutka/SundarGutkaHeader.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactTooltip from 'react-tooltip';
+import { TEXTS, SG_BAANIS, STTM_BLUE } from '@/constants';
+interface ISundarGutkaHeaderProps {
+  sgBaaniLength: string;
+  setSgBaaniLength: (length: string) => {}
+}
+
+export const SundarGutkaHeader: React.FC<ISundarGutkaHeaderProps> = ({
+  sgBaaniLength,
+  setSgBaaniLength,
+}) => {
+  return (
+    <div className="sundarGutkaHeader">
+      <h2>{TEXTS.SUNDAR_GUTKA_HEADER}</h2>
+      <div className="sgBaanis">
+        <span className="sgBaanisInfoIcon" data-tip data-for="sgBaanisInfo">
+          <ReactTooltip backgroundColor={STTM_BLUE} place="top" id="sgBaanisInfo">
+          <span>Different version of Sundar Gutka Baanis</span>
```

Comment:
Does this need to be in a language/parameter file?

Reply:
@saintsoldierx we don't have any language file yet. But yeah eventually we would like to keep all the texts in there